apps/researcher/src/components/tabs.tsx

```diff
@@ -8,6 +8,8 @@ import {locales} from '@/navigation';
 export default function Tabs() {
   const t = useTranslations('Tabs');
 
+  // This header is removed, if we reintegrate this component,
```

Comment:
Perhaps clarify this comment just a little bit?
1. What is 'is removed'? For example: does Next remove this header?
2. What does 'if we reintegrate...' mean? For example: why don't we use this component, and when will we?